hypervisor/debug/uart16550.c

```diff
@@ -134,7 +140,9 @@ void uart16550_init(bool eraly_boot)
 		if (!serial_port_mapped) {
 			serial_pci_bdf.value = get_pci_bdf_value(pci_bdf_info);
 			/* Here will be called before init_paging when vaddr2paddr is identical mapping */
-			uart_base_address = pci_pdev_read_cfg(serial_pci_bdf, pci_bar_offset(0), 4U) & PCIM_BAR_MEM_BASE;
+			uart_base_address.mmio_base_vaddr = pci_pdev_read_cfg(serial_pci_bdf, pci_bar_offset(0), 4U) & PCIM_BAR_MEM_BASE;
```

Comment:
We need to do the PA to VA conversion here.  hpa2hva_early

Reply:
I want to see this patch be merged first

Reply:
OK